docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,190 @@ description: guide for doing a lift and shift migration of airflow to dagster
+   ```yaml
+   locations:
+     - location_name: dagster_migration
+       code_source:
+         python_file: dagster_migration.py
+   ```
 
-- **Using [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) or [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets)** are the easiest methods for defining permissions.
+3. Run the following to add the project to your Dagster Cloud deployment as a code location:
 
-- **If your Airflow DAGs used [`KubernetesPodOperators`](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html)**, it's possible that you used the `in_cluster` config or loading a `kube_config` file. When migrating, we recommend switching to [using connections with a `kube_config` JSON blob](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/connections/kubernetes.html) to make things easier.
+   ```bash
+   dagster-cloud workspace add-location --from ./dagster_cloud.yaml
+   ```
 
 ---
 
-## Limitations
+## Step 7: Migrate permissions to Dagster
 
-There are a few limitations to converting Airflow DAGs into Dagster Jobs, Assets, and Schedules. The following features are currently unsupported:
+Your Airflow instance likely had specific IAM or Kubernetes permissions that allowed it to successfully run your Airflow DAGs. To run the migrated Dagster jobs, you'll need to duplicate these permissions for Dagster.
 
-- `retry-from-failure` in Dagster
-- Using `prev_execution_date` in Airflow-templated DAGs
-- `SubDAGOperators`
-- Airflow Datasets
+- **We recommend using [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) or [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets)** to define permissions whenever possible.
 
-Most of the current limitations are due to a locally-scoped, ephemeral Airflow database being created for each run. If your DAGs implicitly rely on Airflow database state across `DagRuns`, then the current migration tooling won't work for you.
+- **If you're unable to use Airflow connections or environment variables,** you can attach permissions directly to the infrastructure where you're deploying Dagster.
+
+- **If your Airflow DAGs used [`KubernetesPodOperators`](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html)**, it's possible that you loaded a `kube_config` file or used the `in_cluster` config. When migrating, we recommend switching to [using connections with a `kube_config` JSON blob](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/connections/kubernetes.html) to make things easier.
+
+---
+
+## Containerized Operator Considerations
```

Comment:
❓ Is this relevant to a specific step in this guide, or just migration in general? If it's specific to a step (perhaps the local dev portion of the guide?), consider moving it there. A user might not see it otherwise, or see it after the fact and have a bad time.

LMK your thoughts - I didn't edit this because I wanted info first, but I'd like to take a pass at it before this gets merged.

Reply:
yeah let me move this up into the running dagit sectioin